taxi/views.py

```diff
@@ -112,7 +111,7 @@ class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
 def assign_or_delete_car(request, pk):
     car = get_object_or_404(Car, id=pk)
     driver = request.user
-    if car in driver.cars.all():
+    if driver.cars.filter(id=pk).exist():
```

Comment:
queryset doesn't have exist method :)

Reply:
I was inattentive, but after checking, everything is working now.